src/app/pages/ArticlePage/experimentTopStories/helpers.tsx

```diff
@@ -85,12 +85,13 @@ export const getExperimentTopStories = ({
     id,
   });
 
-  const transformedBlocks = shouldEnableExperimentTopStories
-    ? insertExperimentTopStories({
-        blocks,
-        topStoriesContent,
-      })
-    : blocks;
+  const transformedBlocks =
+    shouldEnableExperimentTopStories && blocks.length > 2
```

Comment:
Added the `blocks.length > 2` condition to account for:
- The current formula for calculating the halfway position placing the experiment top stories block before the original block if there is only one block in the array.
- The experiment top stories block being placed before any content if there were only two blocks (e.g. headline block + image block).